### ISSUE TITLE
Youtube Live Issues on VLC and VLC.net

### ISSUE THREAD

**Issue by aah50** [ZeBobo5/Vlc.DotNet#487]
I have an issue about Vlc.DotNet.

# Generic information

- **Vlc.DotNet version** : 3.0.0.4
- **Vlc.DotNet project used** : WPF (but issue is across all)
- **libvlc version** : x86 and x64 got it from FTP
- **.net version** : 4.5.1
- **Project language** : C#
- **Project build architecture** : All (x86/x64 etc)
- **Operating system** : Windows .x64

# Summary

The issue is happening with opening "Network Stream" and using a Youtube Live video (H.264 format or the like). All that is shown is a blank screen for a good 5 minutes (time may vary). The playing of the stream suddenly starts to work. The same issue happens with the latest VLC version (also on older versions)
example streams:
- https://www.youtube.com/v/dp8PhLsUcFE (or https://www.youtube.com/watch?v=dp8PhLsUcFE) starts playing after an hour or so.
- https://www.youtube.com/v/h8rttB7NIyY starts playing after 5 mins or so

- [] I confirm that my issue doesn't happen in VLC itself.

**Comment by mfkl** [comment]
> https://www.youtube.com/v/h8rttB7NIyY starts playing after 5 mins or so

This works instantly here, how good is your internet connection?

> libvlc version : x86 and x64 got it from FTP

What *version* is it though?

**Comment by aah50** [comment]
Updated the VLC (3.0.0.4) and VLC .net version (3.0.0-develop317).
My internet connection opens these streams well enough through regular browsers (4Mbit). Does playing with VLC require higher speeds?

**Comment by jeremyVignelles** [comment]
3.0.0.4 or 3.0.4? Did you try with VLC itself?

**Comment by aah50** [comment]
Yes, the issue I am describing is happening on VLC as well (V3.0.4 and even older versions both x86 and x64)
